e2e_tests/tests/command/command.py

```diff
@@ -24,8 +24,10 @@ def __init__(self, process: subprocess.Popen, detach: bool = False):
         else:
             iterator = iter(self.process.stdout)  # type: ignore
             line = next(iterator)
-            m = re.search(rb"Scheduling .* \(id: (.*)\)", line)
-            assert m is not None
+            m = None
+            while line and not m:
```

Comment:
Ultimately no, with the updates here if all lines are exhausted and `m` is still now we will return `None` and not a valid task_id so all tests that would have failed previously will still fail now.

Reply:
yeah, but, shouldn't we explicitly fail like we used to instead of relying on something downstream to fail because task_id is None, or am I missing something?

Reply:
Sorry I was confused,  I missed that I removed the assertion it has been re added.